plugins/by-name/clipboard-image/default.nix

```diff
@@ -95,17 +95,18 @@ in
       ];
     };
 
-    clipboardPackage = mkOption {
-      type = with types; nullOr package;
-      description = ''
-        Which clipboard provider to use.
+    clipboardPackage = lib.mkPackageOption pkgs "clipboard privider" {
+      nullable = true;
+      default = null;
+      example = [ "wl-clipboard" ];
+      extraDescription = ''
+        ${"\n\n"}
 
         Recommended:
           - X11: `pkgs.xclip`
           - Wayland: `pkgs.wl-clipboard`
           - MacOS: `pkgs.pngpaste`
       '';
-      example = pkgs.wl-clipboard;
     };
```

Comment:
Was it intended that this option not have a default? I don't see any reason why we'd want that. Defaulting to null makes sense to me.

Reply:
I seems like the plugin _requires_ a package, else it raises a checkhealth issue:

```
local packages = {
  x11 = { name = "xclip", binary = "xclip" },
  tty = { name = "xclip", binary = "xclip" },
  wayland = { name = "wl-clipboard", binary = "wl-paste" },
  darwin = { name = "pngpaste", binary = "pngpaste" },
  windows = { name = "powershell", binary = "powershell.exe" },
  wsl = { name = "powershell", binary = "powershell.exe" },
}
```

Though we should maybe add a warning if it's a default prio null instead of not providing a value

Reply:
I'd handle this in another PR. Would be hard to default it sanely so should probably be some kind of warning/assertion that it needs to be configured if the plugin doesn't check itself.

Reply:
No issue with that